spot-price-monitor.py

```diff
@@ -1,10 +1,14 @@
 import argparse
 from kubernetes import client, config
 import boto3
+import requests
+import re
+import json
 from botocore.exceptions import ClientError
 from prometheus_client import start_http_server, Gauge, Counter
 from datetime import datetime
-from time import sleep
+import time
+import logging
```

Comment:
If you are using the logging framework I wonder if the standard isn't

```
import logging
log = logging.getLogger(__name__)
```

Reply:
I agree with @dobesv but since this is internal it can be changed at a later date